packages/suite/src/views/start/SuiteStart.tsx

```diff
@@ -0,0 +1,76 @@
+import { ReactNode } from 'react';
+import styled from 'styled-components';
+
+import { analytics } from '@trezor/suite-analytics';
+import { DOCS_ANALYTICS_URL, DATA_TOS_URL } from '@trezor/urls';
+import { DataAnalytics } from '@trezor/components';
+import { selectIsAnalyticsConfirmed } from '@suite-common/analytics';
+
+import { useDispatch, useSelector } from 'src/hooks/suite';
+import { PrerequisitesGuide, TrezorLink, WelcomeLayout } from 'src/components/suite';
+import { selectPrerequisite } from 'src/reducers/suite/suiteReducer';
+import { rerun } from 'src/actions/recovery/recoveryActions';
+import { SecurityCheck } from 'src/views/onboarding/steps/SecurityCheck/SecurityCheck';
+import { selectDevice } from '@suite-common/wallet-core';
+
+const Content = styled.div`
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+`;
+
+const StyledTrezorLink = styled(TrezorLink)`
+    color: ${({ theme }) => theme.TYPE_LIGHT_GREY};
+`;
+
+export const SuiteStart = () => {
+    const confirmed = useSelector(selectIsAnalyticsConfirmed);
+    const recovery = useSelector(state => state.recovery);
+    const prerequisite = useSelector(selectPrerequisite);
+    const device = useSelector(selectDevice);
+
+    const dispatch = useDispatch();
+
+    const onConfirm = (trackingEnabled: boolean) => {
+        if (trackingEnabled) {
+            analytics.enable();
+        } else {
+            analytics.disable();
+        }
+        if (recovery.status === 'in-progress') {
+            // T2T1 remember the recovery state and should continue with recovery
+            dispatch(rerun());
+        }
+    };
+
+    let component;
+
+    if (!confirmed) {
+        // 1st substep
+        component = (
+            <DataAnalytics
+                onConfirm={onConfirm}
+                analyticsLink={(chunks: ReactNode[]) => (
+                    <StyledTrezorLink variant="underline" href={DOCS_ANALYTICS_URL}>
+                        {chunks}
+                    </StyledTrezorLink>
+                )}
+                tosLink={(chunks: ReactNode[]) => (
+                    <StyledTrezorLink variant="underline" href={DATA_TOS_URL}>
+                        {chunks}
+                    </StyledTrezorLink>
+                )}
+            />
+        );
+    } else if (device) {
+        component = <SecurityCheck />;
+    } else if (prerequisite) {
+        component = <PrerequisitesGuide />;
+    }
```

Comment:
nitpick, codestyle: I would prefer to extract this to separate component that can return on of those three components based on the condition. But probably the only difference would be readability.

Reply:
18763e644593bbcefa895d1658c4ff6034bbfcaf